sonar-orchestrator/src/main/java/com/sonar/orchestrator/build/BuildResult.java

```diff
@@ -36,20 +39,37 @@ public String getLogs() {
     return logs.toString();
   }
 
+  /**
+   * @deprecated since 3.13 use {@link #getLastStatus()}
+   */
+  @Deprecated
   public Integer getStatus() {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)